docsrc/imap/download/release-notes/index.rst

```diff
@@ -31,6 +31,16 @@ Latest development snapshot is |imap_development_release_notes|. Documentation a
 Supported Product Series
 ========================
 
+Series 3.9
+----------
+
+Documentation at :cyrus-3.9:`/`.
+
+.. toctree::
+    :maxdepth: 1
+
+    3.9/index
+
```

Comment:
3.9 is the current unstable version, not a "Supported Product Series".  There is no `3.9/` tree on the website.  Please just drop this whole commit.